sdk/highlight-go/highlight.go

```diff
@@ -13,6 +13,7 @@ import (
 	"github.com/pkg/errors"
 	"go.opentelemetry.io/otel/attribute"
 	semconv "go.opentelemetry.io/otel/semconv/v1.4.0"
+	"gorm.io/gorm"
```

Comment:
does this mean gorm is a dependency of our SDK now?

Reply:
Ah, yeah, I guess it would be. I see your comment about putting everything in the `highlight` package, so I guess that would mean we need to make GORM, Logrus, etc. deps of this now, right?

Reply:
https://github.com/highlight/highlight/blob/ad309bf27a6b7c4920e8cd091e6813f3ae098f9c/sdk/highlight-go/go.mod#L1-L20

actually it doesn't matter whether these are in the same package or not because they will be part of the same module and thus share dependencies. i'm just worried what impact this will have if a customer on gorm v2 uses our sdk depending on v1, or vice versa, but maybe that's something we can test? what happens with our backend (on gorm v1) if we make this sdk use gorm v2?